cdisc_rules_engine/services/__init__.py

```diff
@@ -1,4 +1,4 @@
 from cdisc_rules_engine.config import config
-from cdisc_rules_engine.services.logging_service_factory import LoggingServiceFactory
+from cdisc_rules_engine.services.logging import LoggingServiceFactory
 
-logger = LoggingServiceFactory.get_logger(config)
+logger = LoggingServiceFactory(config).get_service()
```

Comment:
We use this logger in the run_validation.py file. In that file we have the option to disable the logger. https://github.com/cdisc-org/cdisc-rules-engine/blob/development/scripts/run_validation.py#L126 Can we add a method to the interface and logger for disabling logging?

Reply:
Updated, implemented the same interface as logging library provides:
![image](https://user-images.githubusercontent.com/51370006/198042192-b6d4e4ff-19e6-40e3-8fb7-bd271a7e04ff.png)